frontend/src/ui/Input.tsx

```diff
@@ -36,19 +36,33 @@ export type TextAreaProps = React.ComponentPropsWithoutRef<"textarea"> & {
 
 /** A styled multi-line text area */
 export const TextArea = React.forwardRef<HTMLTextAreaElement, TextAreaProps>(
-    ({ error = false, ...rest }, ref) => (
-        <textarea
-            ref={ref}
+    ({ error = false, ...rest }, ref) => {
+        const textAreaRef = useRef<HTMLTextAreaElement>(null);
+        useImperativeHandle(ref, () => textAreaRef.current as HTMLTextAreaElement);
+        const initialHeight = textAreaRef.current?.scrollHeight;
+
+        const handleInput = (e: React.ChangeEvent<HTMLTextAreaElement>) => {
+            if (textAreaRef.current) {
+                textAreaRef.current.style.height = "auto";
+                textAreaRef.current.style.height = `${e.target.scrollHeight}px`;
```

Comment:
Do you set this to `auto` first to make `scrollHeight` have the correct value? Like mentioned in this one article? If so, please add a comment, because otherwise the first assignment seems useless.

Reply:
Hm I adapted this solution from [this article](https://dev.to/gabrielmlinassi/how-to-build-a-dynamic-resizable-textarea-38pf) though I made some adjustments, as I found that some things work differently than described there.
In Tobira's case, the `auto` thing is needed for another reason than stated there, but it's not useless.
Without it, the height of the textarea decreases by a small amount whenever a single char is deleted, like: 

https://github.com/elan-ev/tobira/assets/94838646/0880b502-457b-4882-952c-e20614bf3857

Setting this to `auto` fixes this, but I don't know how best to explain this in a comment.